srcpkgs/avahi-discover/template

```diff
@@ -80,7 +80,7 @@ do_install() {
 	mv ${wrksrc}/tmpinstall/usr/include/avahi-ui* ${DESTDIR}/usr/include
 	mv ${wrksrc}/tmpinstall/usr/include/avahi-qt* ${DESTDIR}/usr/include
 	mv ${wrksrc}/tmpinstall/usr/include/avahi-libevent* ${DESTDIR}/usr/include
-	mv ${wrksrc}/tmpinstall/usr/lib/python3.8 ${DESTDIR}/usr/lib
+	mv ${wrksrc}/tmpinstall/${py3_lib} ${DESTDIR}/usr/lib
 	# Python files are compatible with both 2.7 and 3.x, make both subpackages
 	vmkdir ${py2_lib}
```

Comment:
And I think it's time to just drop `avahi-python`, keep `avahi-python3` only.

Reply:
I agree, done.